designs/ClientDevProcess.md

```diff
@@ -0,0 +1,145 @@
+
+## Tracking Private/Sensitive Issues
+
+This document focuses on **public** issue tracking. Private issues, such as private design specs and security issues are handled through separate processes, and are thus out of the scope of this document.
+
+## The Board
+
+- Link: https://github.com/NuGet/Home#workspaces/nuget-client-team-55aec9a240305cf007585881/board?repos=29996513
+
+### Pipelines
+
+These are the columns on the board, and each one has a specific purpose, described below. Note that epics are handled slightly different, as outlined in [Epic Tracking](#epic-tracking):
+
+#### New Issues
+
+This is the default pipeline/column for incoming issues. Whenever a new issue is created in NuGet/Home, a card is added here to represent that story.
+
+As the team works to whittle down the list of issues in the repository, this pipeline will become empty except for un-triaged issues. From here, one of three things will happen to an issue:
+
+1. It will be moved to the Icebox
+2. It will be moved to the Backlog
+3. It will be closed
+
+#### Icebox
+
+The Icebox is for issues the team thinks are worth doing, doesn't foresee having time to do themselves. Generally, these issues should also be tagged as `help wanted` and, if appropriate, `good first issue`. Issues from the Icebox can progress in two ways:
+
+1. If the Backlog is exhausted
+2. The issue is closed because it's no longer relevant
+
+#### Backlog
+
+The Backlog is the overall task list for the Client Team. This is the work the team intends to be doing itself in the current sprint or in the next two sprints. Anything else goes back in the Icebox.
+
+The general intention with the Backlog is that when a team member has completed all their work, they should be able to pull off something near the top of the Backlog and start working on it.
+
+Once something is in the Backlog, it can also be assigned a Sprint. This allows us to filter the Backlog into a Sprint Backlog by filtering by Sprint! Sprint Backlog items take priority over other Backlog items and should generally be assigned to Sprints during Sprint Planning, sized in such a way that it'll be just enough for 2 weeks of work.
+
+As items enter the Backlog, they should be prioritized as well as possible, with more important issues bubbling to the top, and, if high priority, marked as such, which will pin it to the top.
+
+All issues in the Backlog should be estimated, with each Story Point being roughly 1 hour, with a strong preference towards increments of 5. Estimates include review and review fixes time. Epics should not be directly estimated, and should instead rely on their sub-tasks' estimates.
+
+The Backlog can further be filtered into three main buckets, using GitHub Labels:
+
+1. Quality - test improvements, bugfixes

```

Comment:
I'll rework my labels proposal at some point based on these guidelines :)

Reply:
Yes, thank you!